kymatio/frontend/sklearn_frontend.py

```diff
@@ -0,0 +1,41 @@
+from sklearn.base import BaseEstimator, TransformerMixin
+
+
+class ScatteringTransformerMixin(BaseEstimator, TransformerMixin):
+    def fit(self, x=None, y=None):
```

Comment:
should we decorate those?(I'm simply proposing)

Reply:
Decorate with what?

Reply:
like this: https://github.com/scikit-learn/scikit-learn/blob/3e92edbab8b1d210523e775d2050039055dda1d7/sklearn/pipeline.py#L395

Reply:
Aucune idée. What does that do?

Reply:
high level instruction but I'm not a python/sklearn person

Reply:
Let's say it's fine